src/core/ui/dialogs/ConfirmationDialog.tsx

```
@@ -53,9 +53,13 @@ const ConfirmationDialog = ({ entities, actions, options, state }: ConfirmationD
   return (
     <Dialog open={options.show} aria-labelledby="confirmation-dialog" onClose={actions.onCancel}>
       <DialogHeader onClose={actions.onCancel}>
-        <BlockTitle>{title}</BlockTitle>
+        <BlockTitle>
+          <>{title}</>
+        </BlockTitle>
       </DialogHeader>
-      <DialogContent>{content}</DialogContent>
+      <DialogContent>
+        <>{content}</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)